brewcenter_api/brew_data/tests/test_style_model.py

```diff
@@ -0,0 +1,40 @@
+from django.test import TestCase
+from ..models.style import Style
+
+
+class StyleTest(TestCase):
```

Comment:
A good test should setup and destroy the data it uses, leaving no trace. Think you could make a tearDown method?

Reply:
Will do, is the folder structure ok? for the project or should it be added to the tests.py, I think its better with a test directory to minimize confusion

Reply:
I prefer folder structures rather than a single file.